chat-bots/src/Data/Chat/Utils.hs

```diff
@@ -58,19 +59,25 @@ infixr 9 |*|
 
 type (\/) = Either
 
-infixr 9 \/
+infixr 6 \/
 
 -------------------------------------------------------------------------------
 
 type a \*/ b = Maybe (Either a b)
 
-infixr 9 \*/
+infixr 6 \*/
 
 -------------------------------------------------------------------------------
 
 type a /+\ b = These a b
 
-infixr 9 /+\
+infixr 6 /+\
+
+can :: Maybe a -> Maybe b -> Maybe (a /+\ b)
```

Comment:
Idk that we're using this anymore.

Reply:
were using it here: https://github.com/cofree-coffee/cofree-bot/blob/feature%2Fparsing/chat-bots/src/Data/Chat/Serialization.hs#L52
```
(*|*) :: (Text -> Maybe a) -> (Text -> Maybe b) -> Text -> Maybe (a /+\ b)
(*|*) p1 p2 = these (fmap This) (fmap That) can . liftA2 These p1 p2
```